src/Microsoft.OData.Core/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ServiceCollectionExtensions.cs" company="Microsoft">
+// Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OData.Edm;
+using Microsoft.OData.Json;
+using Microsoft.OData.UriParser;
+
+namespace Microsoft.OData.Core
+{
+    /// <summary>
+    /// Extension methods for <see cref="IServiceCollection"/>.
+    /// </summary>
+    public static class ServiceCollectionExtensions

```

Comment:
This should be named `OData_ServiceCollectionExtensions` so that you can differentiate OData-specific `IServiceCollection` extensions from other extensions using Reflection, given the namespace guidance above.